WordPress/Sniffs/PHP/YodaConditionsSniff.php

```diff
@@ -46,6 +46,8 @@ public function process_token( $stackPtr ) {
 		$beginners   = PHP_CodeSniffer_Tokens::$booleanOperators;
 		$beginners[] = T_IF;
 		$beginners[] = T_ELSEIF;
+		$beginners[] = T_RETURN;
```

Comment:
Some additional tokens which could be added here to prevent unnecessary token walking:
* T_SEMICOLON
* T_OPEN_PARENTHESIS
* T_CASE

Adding the `T_OPEN_PARENTHESIS` allows for removing the `T_IF` and `T_ELSEIF` by recognizing that the condition is within parenthesis, i.e. `$sample = ( true === $something )`, but also `if ( $something === true )`.